components/Badge/Badge.module.css

```diff
@@ -8,7 +8,7 @@
 
 .Badge > img,
 .Badge > svg {
-  fill: currentColor;
+  fill: currentcolor;
```

Comment:
https://css-tricks.com/currentcolor/

I can't find any article explaining why this value can be camelcase or all lowercase, but both works, so you dont need to change it. You can leave this alone, but just for the future.

Reply:
Hmmm there's no anchor with an href of `#case-sensitivity` on this page.

I see this:
<img width="382" alt="Screen Shot 2022-06-09 at 5 06 03 PM" src="https://user-images.githubusercontent.com/9523719/172880193-5599466c-b00a-4767-9efd-77c799ad3b5f.png">

but that's not related to case sensitivity of CSS _values_.

Reply:
eslint is complaining about it in several places, I think my editor just picked that up. Its on its own commit, we can toss it out and try something else or accept the rule as is. More that husky blocks pushing changes because of such a minor non-issue per html and css4 selectors spec.

Reply:
ESLint is not complaining about `currentColor`.

Reply:
➜  front-end git:(feature/svg-logo) eslint -v
v8.16.0

Reply:
The lockfile specifies: eslint@^8.1.0

So it is indeed because you're using the wrong version. Something happened that affected your `yarn install`.

Reply:
https://snyk.io/test/npm/eslint/0.8.1

As I said before this is a vunerable version and needs to be updated. If you want to throw out the commit with these lowecase changes thats fine but there is obviously some issues with eslint that are outside the scope of the PR. I'm just trying to get through the gates.